app/cloudbuild.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - name: node:10.16.3
+    entrypoint: yarn
+    args: [install, --production]
+  - name: node:10.16.3
+    entrypoint: yarn
+    args: [build]
+    env:
+      - REACT_APP_HTTP_URI=https://graphql-dot-keen-apogee-250800.uc.r.appspot.com
```

Comment:
This should probably be passed in as a [substitution][] so that it can be used in multiple environments.

[substitution]: https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values

Reply:
Oh yeah! That's exactly what I was looking for. This is great!